prepare_data/src/test/groovy/org/orbisgis/common/InputDataFormattingTest.groovy

```diff
@@ -23,21 +23,38 @@ class InputDataFormattingTest {
     @Test
     @EnabledIfSystemProperty(named = "test.bdtopo", matches = "true")
     void inputDataFormatting(){
```

Comment:
The test is a good way but we must test also if there are some rows in the formatted tables

Reply:
Yes, it's in the pipe. 
We are currently working to finalize all the cables and to check if tables are well created.
Then we will check if they are empty.